WebApp/autoreduce_webapp/selenium_tests/pages/configure_new_runs_page.py

```diff
@@ -19,6 +19,8 @@ class ConfigureNewRunsPage(Page, RerunFormMixin, NavbarMixin, FooterMixin, TourM
     """
     Page model class for run summary page
     """
+
+    # pylint:disable=too-many-arguments
```

Comment:
5 is too many?! Guess it counts self, but still 6?!

Reply:
It's funny how everyones opinion differs I am in the 2 is pushing it camp 😄